package/gluon-status-page-api/luasrc/lib/gluon/status-page/www/cgi-bin/interfaces

```diff
@@ -1,27 +1,26 @@
 #!/usr/bin/lua
 
+lutil = require 'luci.util'
 util = require 'gluon.util'
-
 fs = require 'nixio.fs'
 json = require 'luci.jsonc'
+ubus = require 'ubus'
 
 io.write("Access-Control-Allow-Origin: *\n")
 io.write("Content-type: application/json\n\n")
 
-f = io.popen('batctl if')
-
+local uconn = ubus.connect()
+if not uconn then
+	error("Failed to connect to ubusd")
+end
+list = util.get_mesh_devices(uconn)
+ubus.close(uconn)
 interfaces = {}
-
-for line in f:lines() do
-  ifname = line:match('^(.-):')
-  if ifname ~= nil then
+for _,ifname in ipairs(list) do
     pcall(function()
-      local address = util.trim(fs.readfile('/sys/class/net/' .. ifname .. '/address'))
+      local address = lutil.trim(fs.readfile('/sys/class/net/' .. ifname .. '/address'))
```

Comment:
Why does this change *util* to *lutil*? The usage of `luci.util` can be avoided altogether.